File: Builder/python/Computer.py
class Computer:
    cpu: str
    ram: str
    storage: str
    
    def __init__(self) -> None:
        self.cpu = ''
        self.ram = ''
        self.storage = ''
    
    def set_cpu(self, cpu: str) -> None:
        self.cpu = cpu
        
    def set_ram(self, ram: str) -> None:
        self.ram = ram
        
    def set_storage(self, storage: str) -> None:
        self.storage = storage
        
    def display_info(self) -> None:
        print("Computer configuration:")
        print(f"CPU: {self.cpu}")
        print(f"RAM: {self.ram}")
        print(f"Storage: {self.storage}")
        

File: Builder/ts/Computer.ts
export class Computer {
    cpu: string;
    ram: string;
    storage: string;

    constructor() {
        this.cpu = '';
        this.ram = '';
        this.storage = '';
    }

    set_cpu(cpu: string): void {
        this.cpu = cpu;
    }

    set_ram(ram: string): void {
        this.ram = ram;
    }

    set_storage(storage: string): void {
        this.storage = storage;
    }

    display_info(): void {
        console.log('Computer configuration:');
        console.log(`CPU: ${this.cpu}`);
        console.log(`RAM: ${this.ram}`);
        console.log(`Storage: ${this.storage}`);
    }
}